neutron/assetlist.json

```diff
@@ -528,6 +528,32 @@
       "images": [
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/bad.png"
+        },
```

Comment:
The schema has broken here. You need all the appropriate closing brackets for the previous asset and its properties before starting a new asset.